libraries/AP_Compass/Compass_learn.cpp

```diff
@@ -246,6 +248,16 @@ void CompassLearn::process_sample(const struct sample &s)
     best_error = bestv;
     worst_error = worstv;
     best_yaw_deg = wrap_360(degrees(s.attitude.z) + besti * (360/num_sectors));
+
+    // send current learn state to gcs
+    uint32_t now = AP_HAL::millis();
+    if (!converged && now - last_learn_ts >= 2000) {
+        gcs().send_text(MAV_SEVERITY_INFO, "CompassLearn: %u>%d %.1f<%d %.1f>%d",
+                                                (unsigned)num_samples, COMPASS_LEARN_NUM_SAMPLES,
+                                                best_error,COMPASS_LEARN_BEST_ERROR_THRESHOLD,
```

Comment:
Intenting needs fixing.
```suggestion
                                                (unsigned)num_samples,
                                                COMPASS_LEARN_NUM_SAMPLES,
                                                best_error,
                                                COMPASS_LEARN_BEST_ERROR_THRESHOLD,
```